plugins/SitesManager/API.php

```diff
@@ -1515,20 +1515,46 @@ public function renameGroup($oldGroupName, $newGroupName)
         return true;
     }
 
-    public function getPatternMatchSites($pattern)
+    /**
+     * Find websites matching the given pattern.
+     *
+     * Any website will be returned that matches the pattern in the name, URL or group.
+     * To limit the number of returned sites you can either specify `filter_limit` as usual or `limit` which is
+     * faster.
+     *
+     * @param string $pattern
+     * @param int|false $limit
+     * @return array
+     */
+    public function getPatternMatchSites($pattern, $limit = false)
```

Comment:
Is it possible to just handle `filter_limit` explicitly instead of having two parameters (filter_limit & limit)?